include/godot_cpp/core/class_db.hpp

```diff
@@ -82,23 +83,19 @@ class ClassDB {
 		const char *parent_name = nullptr;
 		GDNativeInitializationLevel level = GDNATIVE_INITIALIZATION_SCENE;
 		std::unordered_map<std::string, MethodBind *> method_map;
-		std::unordered_map<std::string, MethodInfo> signal_map;
-		std::list<MethodBind *> method_order;
-		GDExtensionClassInstancePtr (*constructor)(void *data);
+		std::set<std::string> signal_names;
```

Comment:
Why we use STL classes here instead of Godot's with custom allocators?

Reply:
Because the Godot classes aren't natively implemented in godot-cpp so you need to go over the API barrier, seems wasteful.
Just using STL in these simple situations does the trick without any complexity.

Reply:
Thanks for the info!